vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -755,7 +755,7 @@ EXPORT(void, sceGxmFinish, SceGxmContext *context) {
     assert(context);
 
     if (!context)
-        RET_ERROR(SCE_GXM_ERROR_INVALID_POINTER);
+        return;
```

Comment:
return nothing normal ?

Reply:
The return type of the function is `void` so yes, it is normal

Reply:
oh, i see lol